docker/wait.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+
+# Wait for Minio
+until curl -s http://localhost:8123
+do
+  echo 'Waiting for Clickhouse...'
+  sleep 4
```

Comment:
What about the second Clickhouse instance?

Reply:
sry, I forgot